frontend/templates/product-list/index.tsx

```diff
@@ -79,10 +80,12 @@ export const getProductListServerSideProps = ({
       let shouldRedirectToCanonical = false;
       let canonicalPath: string | null = null;
 
+      const ifixitOrigin = ifixitOriginFromHost(context.req.headers.host ?? '');
```

Comment:
I think this will always be `*.vercel.app` because the host header *has* to be set to one of our app's domains so vercel knows which app to send the request to. However, the original host (that the browser requested) is often available on a different header

Reply:
Cool I'll check the forwarded host like you suggest below.